frontend/src/pages/modelServing/screens/projects/kServeModal/ManageKServeModal.tsx

```diff
@@ -116,6 +131,11 @@ const ManageKServeModal: React.FC<ManageKServeModalProps> = ({
     }
   }, [currentProjectName, setCreateDataInferenceService, isOpen]);
 
+  // Refresh model format selection when changing serving runtime template selection
+  React.useEffect(() => {
+    setCreateDataInferenceService('format', { name: '' });
+  }, [createDataServingRuntime.servingRuntimeTemplateName, setCreateDataInferenceService]);
+
```

Comment:
Fixed a small bug here. When selecting a new template, it will reset the model frame selection now (it didn't before). Not sure if there is a ticket logged.

Reply:
I can't find a ticket, I think it's probably ok to just fix this here. @lucferbux do you happen to know if there's a bug logged for this?